lib/testing/src/lib/process-services-cloud/pages/people-cloud-component.page.ts

```diff
@@ -65,6 +66,11 @@ export class PeopleCloudComponentPage {
         return this.assigneeChipList.all(by.css('mat-chip')).first().getText();
     }
 
+    async getChipAssigneeNo(): Promise<number> {
```

Comment:
no = number ? total?

Reply:
no means number

Reply:
let's avoid abbreviation when can create confusion

Reply:
No it's the official abbreviation for number, i don't think it can create confusion. Anyway, i don't mind changing the name, but please say clear what name doesn't create confusion.

Reply:
getChipAssigneeCount is IMO a better option